scripts/stop.sh

```diff
@@ -3,4 +3,8 @@
 ps aux | grep ksql-server-start | grep -v "grep ksql" | awk '{print $2}' | xargs kill -15
 
 docker-compose down
-docker volume ls -q --filter dangling=true | xargs docker volume rm
+
+volumes=$(docker volume ls -q --filter="dangling=true")
+if [ -n "$volumes" ]; then
+	docker volumes rm "$volumes"
```

Comment:
* Command is `docker volume`, not `docker volumes`
* `docker volume` returns a list, so it needs to iterate through each item in the list

Consider changing code to:

```
volumes=$(docker volume ls -q --filter="dangling=true")
if [ -n "$volumes" ]; then
  docker volume ls -q --filter dangling=true | xargs docker volume rm
fi
```